src/site/content/en/blog/optimize-lcp/index.md

```diff
@@ -36,11 +36,8 @@ viewport becomes visible. It can be used to determine when the main content of t
 rendering on the screen.
 
 <picture>
-  <source srcset="../vitals/lcp_8x2.svg" media="(min-width: 640px)">
-  <img class="w-screenshot w-screenshot--filled"
-      src="../vitals/lcp_4x3.svg"
-      alt="Good LCP values are 2.5 seconds, poor values are greater than 4.0
-            seconds and anything in between needs improvement">
+  <source srcset="{{ "image/tcFciHGuF3MxnTr1y5ue01OGLBn2/9mWVASbWDLzdBUpVcjE1.svg" | imgix }}" media="(min-width: 640px)">
```

Comment:
Double check this renders the same on both sites since they're using picture to crop the image